specification/containerservice/resource-manager/readme.java.md

```diff
@@ -38,6 +38,20 @@ batch:
   - tag: package-2021-05
```

Comment:
also need to update readme.md, refer to the previous APIs

Reply:
Weird. I did update readme.md with below content but it does not exist in this PR.
```

### Tag: package-2021-09

These settings apply only when `--tag=package-2021-09` is specified on the command line.

```yaml $(tag) == 'package-2021-09'
input-file:
  - Microsoft.ContainerService/stable/2021-09-01/managedClusters.json
```
You can see https://github.com/AbelHu/azure-rest-api-specs/commit/34d6e83e8145836bc31e342b7eaed3aa95580765#diff-5edcad5257543ceda29d72e1ff8cfb550ced7ae4e844feca5f1f591214a7ce89

Reply:
Found the reason. The file was automatically updated when creating the branch dev-containerservice-Microsoft.ContainerService-2021-09-01. https://github.com/Azure/azure-rest-api-specs/blob/dev-containerservice-Microsoft.ContainerService-2021-09-01/specification/containerservice/resource-manager/readme.md

Reply:
Though the readme.md was automatically updated when creating the branch 2021-09-01 branch, we still need update it with a few changes, see my commit https://github.com/Azure/azure-rest-api-specs/pull/15476 for reference.